ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp

```diff
@@ -305,7 +305,7 @@ void string_to_spv(const std::string& _name, const std::string& in_fname, const
     {
         // wait until fewer than N compiles are in progress.
         // 16 is an arbitrary limit, the goal is to avoid "failed to create pipe" errors.
-        uint32_t N = 16;
+        uint32_t N = (std::max)(1u, (std::min)(16u, std::thread::hardware_concurrency()));
```

Comment:
```suggestion
        uint32_t N = std::max(1u, std::min(16u, std::thread::hardware_concurrency()));
```
What's with the brackets?

Reply:
It's a way of dealing with windows.h defining min and max as macros. The other solution is to #define NOMINMAX before including it.